pkg/config/fulcio_config_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//go:build !hermetic
+
+package config
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"gopkg.in/yaml.v3"
+)
+
+type FulcioConfigMap struct {
+	Data map[string]string `yaml:"data,omitempty"`
+}
+
+// It tests that the config/fulcio-config.yaml is properly parsable
```

Comment:
Can we make this comment more precise, that this is a CI test for verifying that the configuration to be used by the public good instance is valid? (it's doing more than verifying that it's parseable)

Reply:
yep! it makes a lot of sense. on it